__tests__/app.test.js

```diff
@@ -241,3 +241,68 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201 - responds with the posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "I never tire of learning about mitch",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toEqual(
+          expect.objectContaining({
```

Comment:
This is great. Nice and precise where possible, leaving flexibility for the comment id. Very good!